resources/js/src/app/components/itemList/filter/ItemRangeSlider.spec.ts

```diff
@@ -0,0 +1,170 @@
+                    items: [],
+                    name:'59 - 99',
+                    position: 'item',
+                    selected: false
+                },
+                {
+                    count: 1,
+                    id: '21',
+                    image: '',
+                    items: [],
+                    name: '119 - 149',
+                    position: 'item',
+                    selected: false
+                }
+            ]
+        };
+
+        store.state.itemList = {
+            isLoading: false,
+            selectedFacets: [],
+            facets: [facet]
+        };
+        UrlBuilder.updateSelectedFilters = jest.fn();
+
+        const wrapper = shallowMount(
+            ItemRangeSlider,
+            {
+                propsData: { facet },
+                stubs: {'icon': true},
+                directives: {
+                    tooltip() {}
+                },
+                store,
+                localVue
+            }
+        );
+
+        const submitButton = wrapper.find('button');
+        await submitButton.trigger('click')
+        expect(UrlBuilder.updateSelectedFilters).toHaveBeenNthCalledWith(1, facet, 'price', {'max': 149, 'min': 59});
+
+        wrapper.setData({valueFrom: 100, valueTo: 140.55});
+        await submitButton.trigger('click')
+        expect(UrlBuilder.updateSelectedFilters).toHaveBeenNthCalledWith(2, facet, 'price', {'max': 140.55, 'min': 100});
+    });
+
+    it('Disables the filter submit button if fields contain incorrect values', async () => {
+        const facet: Facet = {
+            cssClass: '',
+            findologicFilterType: 'range-slider',
+            id: 'price',
+            isMain: true,
+            itemCount: 0,
+            maxValue: 149,
+            minValue: 59,
+            name: 'Price',
+            noAvailableFiltersText: '',
+            select: 'single',
+            step: 0.1,
+            type: '',
+            unit: '£',
+            values: [
+                {
+                    count: 9,
+                    id: '20',
+                    image: '',
+                    items: [],
+                    name:'59 - 99',
+                    position: 'item',
+                    selected: false
+                },
+                {
+                    count: 1,
+                    id: '21',
+                    image: '',
+                    items: [],
+                    name: '119 - 149',
+                    position: 'item',
+                    selected: false
+                }
+            ]
+        };
+
+        store.state.itemList = {
+            isLoading: false,
+            selectedFacets: [],
+            facets: [facet]
+        };
+        UrlBuilder.updateSelectedFilters = jest.fn();
+
+        const wrapper = shallowMount(
+            ItemRangeSlider,
+            {
+                propsData: { facet },
+                stubs: {'icon': true},
+                directives: {
+                    tooltip() {}
```

Comment:
You probably want to use `jest.fn()` instead of an empty method.
```suggestion
                    tooltip: jest.fn
```

We don't use it, but it's better than having an empty method.